frontend/src/pages/ChargepointPage.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components/macro";
+import ChargepointItem from "../components/ChargepointItem";
+import UseChargepoints from "../hooks/UseChargepoints";
+
+export default function ChargepointPage() {
+  const { chargepoints } = UseChargepoints();
+
+  console.log(chargepoints); // Output on console for testing
+
+  return (
+    <Wrapper>
+      {chargepoints.map((chargepoint) => (
+        <ChargepointItem key={chargepoint.id} chargepoint={chargepoint} />
+      ))}
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  ul {
```

Comment:
`ChargePointItem` neither contains an ul nor a li, so these styles can be deleted.